client/service-container-chain/src/service.rs

```diff
@@ -322,13 +314,35 @@ fn start_consensus_container(
     force_authoring: bool,
     relay_chain_slot_duration: Duration,
     para_id: ParaId,
-    orchestrator_para_id: ParaId,
-    collator_key: CollatorPair,
     overseer_handle: OverseerHandle,
     announce_block: Arc<dyn Fn(Hash, Option<Vec<u8>>) + Send + Sync>,
 ) {
-    let slot_duration = cumulus_client_consensus_aura::slot_duration(&*orchestrator_client)
-        .expect("start_consensus_container: slot duration should exist");
+    let crate::spawner::CollationParams {
+        collator_key,
+        orchestrator_tx_pool,
+        orchestrator_client,
+        orchestrator_para_id,
+        solochain,
+    } = collation_params;
+    let slot_duration = if solochain {
+        // TODO: always use 6s for aura, not sure why I was getting an error when trying to use
```

Comment:
remove this TODO, aura is not implemented in the relay chain (babe being used)

Reply:
*fixed*